packages/hash/shared/src/entityStore.ts

```diff
@@ -24,7 +24,9 @@ export type DraftEntity<Type extends EntityStoreType = EntityStoreType> = {
   //  keep a dict of entity ids to draft ids, and vice versa
   draftId: string;
 
-  entityVersionCreatedAt: string;
+  // we probably don't need entityVersionCreatedAt
+  entityVersionCreatedAt?: string;
```

Comment:
We probably might not need this, since we can use `updateAt` instead

Reply:
It's no longer needed. Will remove

Reply:
Removed here => https://github.com/hashintel/hash/pull/506/commits/d621dfea38be21558acab396774a3564ab4e32e6